docs/guides/connecting_to_your_data/components/next_steps.md

```diff
@@ -1,5 +1,5 @@
 Now that you've connected to your data, you'll want to work on these **core skills**:
 
-- [How to create a Batch of data from an in-memory Spark or Pandas dataframe](/docs/guides/connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_or_pandas_dataframe)
-- [How to create and edit Expectations with instant feedback from a sample Batch of data](/docs/guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data)
-- [How to create and edit Expectations based on domain knowledge, without inspecting data directly](/docs/guides/expectations/how_to_create_and_edit_expectations_based_on_domain_knowledge_without_inspecting_data_directly)
+- [How to create a Batch of data from an in-memory Spark or Pandas dataframe](../../connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_or_pandas_dataframe.md)
```

Comment:
`../connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_or_pandas_dataframe.md`

Reply:
I couldn't get suggested path to work.